pageserver/src/tenant/size.rs

```diff
@@ -118,6 +118,9 @@ pub(super) async fn gather_inputs(
     ctx: &RequestContext,
 ) -> anyhow::Result<ModelInputs> {
     // refresh is needed to update gc related pitr_cutoff and horizon_cutoff
+    //
+    // FIXME: if a single timeline is deleted while refresh gc info is ongoing, we will fail the
+    // whole computation. It does not make sense from the billing perspective.
```

Comment:
Would like to see a link to the bug (in Selected category) in this comment.

Reply:
That's just a guess, assuming refresh_gc_info is still a problem. Could be it's not.